README.md

```diff
@@ -1,24 +1,26 @@
 # TreasureHuntGame
 
 Business Rules:
-- Game consist of 10 turns
-- It is a single player game.
-- At the beginning of the game the size of the board is determined by the player, board size
+- Game consists of 10 turns
+- It is a single-player game.
+- At the beginning of the game the size of the board is determined by the player, the board size
 cannot be less than 5 cells in width and height.
-- After the size is set, two tree objects, a house object, a treasure object, a robot object and a
+- After the size is set, two tree objects, a house object, a treasured object, a robot object, and a
 player object is placed randomly on the board
-- In each turn the application informs the player about its position and asks which x and y position
+- In each turn, the application informs the player about its position and asks which x and y position
 it wants to move
 o If it hits the robot after a move, the player loses the game
-o If it hits a tree or the house, player cannot move and game skips to the next turn after
+o If it hits a tree or the house, the player cannot move, and game skips to the next turn after
 moving the robot
 o If it hits the treasure, the player wins the game
-o After player moves, the robot moves to a location randomly, if the robot hits the treasure,
-the player loses the game. Robot isn’t effected by tree and house objects
-- After each turn the application informs the player if it is getting closer or far to the treasure.
+o After the player moves, the robot moves to a location randomly, if the robot hits the treasure,
```

Comment:
[Sev 3]: Nice work on the grammar corrections, including 'consists of', 'single-player', and 'affected'. Also, there is a possible typo at line 8 - 'treasured object' should probably be 'treasure object'. Please confirm. These corrections greatly enhance the clarity of the documentation.
```
Category: documentation - Severity: medium
Line 4: Various Grammar and Typo Corrections across text lines
```
**_Commented by gpt-4_**